app/views/partials/layout/_navbar.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="container-flouid">
+  <nav class="navbar navbar-expand-lg bg-light shadow-sm p-3 mb-5 bg-body-tertiary rounded">
+    <div class="container">
+      <a class="navbar-brand" href="/">
+      <%= image_tag("monkeyHats.jpg", class: "rounded-circle logo-size mx-auto") %>
```

Comment:
totally is better use images in the `webp` format this bring a better performance in the SEO and in the `lighthouse` tests